src/plugins/data_views/common/types.ts

```diff
@@ -437,7 +438,7 @@ export type FieldSpec = DataViewFieldBase & {
   /**
    * set if field is a TSDB metric field
    */
-  timeSeriesMetric?: 'histogram' | 'summary' | 'gauge' | 'counter';
+  timeSeriesMetric?: estypes.MappingTimeSeriesMetricType;
```

Comment:
For example, running `GET /tracks/_field_caps?fields=location` in dev tools console returns

```
{
  "indices": [
    "tracks"
  ],
  "fields": {
    "location": {
      "geo_point": {
        "type": "geo_point",
        "metadata_field": false,
        "searchable": true,
        "aggregatable": true,
        "time_series_metric": "position"
      }
    }
  }
}
```

Reply:
I expected API responses to be in sync with its types. Thanks for linking to the PR.

Reply:
> It will take time for this change to trickle up to Kibana

Since it can take time it might be worth adding ` | 'position'` here (at least temporary) and run CI once to check that nothing breaks.